content/reliability/docs/application.md

```diff
@@ -49,10 +49,8 @@ spec:
                   operator: In
                   values:
                   - web-server
-              topologyKey: failure-domain.beta.kubernetes.io/zone
+              topologyKey: topology.kubernetes.io/zone
             weight: 100
-        podAntiAffinity:
-          preferredDuringSchedulingIgnoredDuringExecution:
           - podAffinityTerm:
               labelSelector:
```

Comment:
Wouldn’t we need to remove these lines as well?

Reply:
I don't think so, we want to schedule pod first in different zones then in different nodes in those zones, so if we remove these lines then the scheduler might not distribute pods across all nodes in the zone.

Reply:
@realvz can we close this?